src/hub_predtimechart/ptc_schema.py

```diff
@@ -43,6 +43,11 @@
             "description": "text providing any important information users should know",
             "type": "string",
             "minLength": 1
+        },
+        "task_id_text": {
+            "description": "optional mapping of predtimechart task id values to text. keys are task_ids and values are an object whose keys are ptc task_id `value`s and values are ptc task_id `text`s",
+            "type": "object",
+            # todo additional validation: keys are str and values are dict[str:str]
```

Comment:
This is the blind leading the blind so YMMV, but if all objects are key value pairs and all keys are strings, then we can check that all values are strings.

I think we can [use patternProperties](https://json-schema.org/understanding-json-schema/reference/object#patternProperties) to check for this. 

```suggestion
            "patternProperties": {
                "^*$": {"type": "string"}
            } 
```